storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -540,6 +549,18 @@ func (c *PrometheusConverter) addTimeSeriesIfNeeded(lbls []prompb.Label, startTi
 	}
 }
 
+// handleStartTime adds a zero sample 1 millisecond before ts iff startTs == ts.
+func (c *PrometheusConverter) handleStartTime(startTs, ts int64, value float64, labels []prompb.Label, settings Settings) {
+	if !settings.EnableCreatedTimestampZeroIngestion {
+		return
+	}
+	if value != 0 && startTs > 0 && startTs == ts {
```

Comment:
As discussed in slack, we don't care if the value is zero before ingesting `StartTimeUnixNano`. Actually we want to ingest even if its zero so rate() returns something for the first scrape anyway

Reply:
We also don't want to ingest zeros for datapoint types that translate to gauges

Reply:
Fixed.